plume-cli/src/users.rs

```diff
@@ -94,7 +100,17 @@ fn new<'a>(args: &ArgMatches<'a>, conn: &Connection) {
         .value_of("display-name")
         .map(String::from)
         .unwrap_or_else(|| super::ask_for("Display name"));
+
     let admin = args.is_present("admin");
+    let moderator = args.is_present("moderator");
+    let role = if admin {
+        0
+    } else if moderator {
+        1
+    } else {
+        2
+    };
```

Comment:
that means, on this end, we'd just have a (one) `from_str()`

Reply:
Sorry, but… I don't understand. :sweat_smile: Could you try to rephrase, please?